aten/src/TH/THAllocator.cpp

```diff
@@ -19,17 +19,7 @@
 #endif
 /* end of stuff for mapped files */
 
-struct THDefaultAllocator final : public at::Allocator {
-  at::DataPtr allocate(size_t size) const override {
-    auto* ptr = THAlloc(size);
-    return {ptr, ptr, &THFree, at::DeviceType::CPU};
-  }
-  at::DeleterFnPtr raw_deleter() const override {
-    return &THFree;
-  }
-};
-
-static THDefaultAllocator th_default_allocator;
+static DefaultCPUAllocator th_default_allocator;
```

Comment:
I don't think you want this static variable anymore; it's a second copy of the allocator.